auth/authRouter.js

```diff
@@ -0,0 +1,57 @@
+const router = require("express").Router();
+const bcrypt = require("bcryptjs");
+
+const tokenService = require("./tokenService");
+const uModel = require("../users/usersModel");
+
+router.post("/register", async (req, res) => {
+  console.log(req.body);
+  const creds = req.body;
+  const { username, password } = creds;
+
+  if (!username || !password) {
+    return res.status(400).json({
+      message: `Both a username and password are required to register.`
+    });
+  }
+
+  const hash = bcrypt.hashSync(password, 5);
+  req.body.password = hash;
```

Comment:
```suggestion
  creds.password = hash;
```
We might as well do this since you already set req.body to creds